src/Microsoft.Data.SqlClient/tests/ManualTests/SQL/SqlBulkCopyTest/Helpers.cs

```diff
@@ -15,10 +17,16 @@ internal static void ProcessCommandBatch(Type connType, string constr, string[]
             {
                 object[] activatorArgs = new object[1];
                 activatorArgs[0] = constr;
-                using (DbConnection conn = (DbConnection)Activator.CreateInstance(connType, activatorArgs))
+                using (SqlConnection conn = (SqlConnection)Activator.CreateInstance(connType, activatorArgs))
                 {
+                    // Testing in macOS Azure SQL container requires access token authentication
+                    if (!DataTestUtility.AuthenticatingWithoutAccessToken)
```

Comment:
You can add another property in `DataTestUtility` and keep it positive, instead of multiplying a negative property by adding a negative sign, which is a bit tricky to follow.